src/workspace/layouts/WorkspaceLayout/Sidebar.tsx

```diff
@@ -119,10 +119,10 @@ const Sidebar = (props: SidebarProps) => {
               {t("Files")}
             </NavItem>
             <NavItem
-              href={`/workspaces/${encodeURIComponent(workspaceId)}/databases`}
+              href={`/workspaces/${encodeURIComponent(workspaceId)}/database`}
```

Comment:
Better to stay with plurals "/databases"

Reply:
Okay.